fixtures/Functional/config/doctrine_bundle3.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine:
```

Comment:
honestly since the project is fairly stable, I would be fine with release a new minor version bumping to DoctrineBundle3 instead of supporting the 2 versions